chaoscenter/upgrade-agents/control-plane/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # BUILD STAGE
-FROM golang:1.22 AS builder
+FROM golang:1.16 AS builder
```

Comment:
let's not change this version

Reply:
fixed inthe new [pr](https://github.com/litmuschaos/litmus/pull/4740)